Browser_IDE/executionEnvironment_Internal.js

```diff
@@ -406,10 +406,15 @@ window.addEventListener('message', function(m){
         if (m.data.type == "writeFile"){
             FS.writeFile(m.data.path,m.data.data);
         }
+
+        if(m.data.type == "readFile"){
+            FS.readFile(m.data.path);
+        }
```

Comment:
This will simply read the file and then do nothing with the contents.

To return the contents of the file to the main window, you will need to use the temporary callbacks facility, defined in `fallibleMessage.js`.

Reply:
i found why i always display undefine , as when i debug, 
![image](https://github.com/thoth-tech/SplashkitOnline/assets/112596172/d83d6a2c-0088-4867-b38c-4803c1d6be4f)
i found the data from "FS.readFile(m.data.path);" is empty:
here is the cap screen:
![image](https://github.com/thoth-tech/SplashkitOnline/assets/112596172/924b2b21-6d76-477d-bfcd-e60b8b2e3c26)
so if FS.readFile() works, that means splashkit.log is empty? @jessbdeakin